lib/jsdom/browser/Window.js

```diff
@@ -2,6 +2,7 @@
 
 const webIDLConversions = require("webidl-conversions");
 const { CSSStyleDeclaration } = require("cssstyle");
+const { Performance: PerformanceClass } = require("w3c-hr-time");
```

Comment:
I'd call it maybe "RawPerformance" or "W3CHRTimePerformance". Then you can reuse this name (lowercased) for the variable/field name.

Reply:
`RawPerformance` 👍